### PR TITLE
Add week-3-practise folder with students' exercises

### PR DESCRIPTION
**Pull request by martynamaron**
Feedback welcome on editing and author's signature - I just used their GitHub profile URL.

### CODE REVIEW

**Comment by djgrant** [comment]
These are great exercises! So much more interesting that the ones I come up with. I personally don't like author signatures in shared codebases as I think the feeling of shared ownership encourages collaboration. If they could raise PRs themselves that would be fab, and git is great for attributing credit.

**Comment by martynamaron** [comment]
Thanks @djgrant! We've decided to try it then, and ask students to add their own exercises, so watch this space for PRs! Will try to help them out with it if they're having trouble 👍 
I'll close this PR now.

File: week-3-practise/level-1/1-find.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/*
```
Comment (martynamaron) [inline week-3-practise/level-1/1-find.js]: @samiachi

File: week-3-practise/level-1/2-every.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+/*
```
Comment (martynamaron) [inline week-3-practise/level-1/2-every.js]: @samiachi

File: week-3-practise/level-1/3-filter.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
+/*
```
Comment (martynamaron) [inline week-3-practise/level-1/3-filter.js]: @ivinapontes

File: week-3-practise/level-2/1-map.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+/*
```
Comment (martynamaron) [inline week-3-practise/level-2/1-map.js]: @Atayo

File: week-3-practise/level-2/2-map.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+/*
```
Comment (martynamaron) [inline week-3-practise/level-2/2-map.js]: @aldeez988

File: week-3-practise/level-2/3-filter.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
+/*
```
Comment (martynamaron) [inline week-3-practise/level-2/3-filter.js]: @samiachi

File: week-3-practise/level-3/1-practise-all.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
+/* Password Validation
```
Comment (martynamaron) [inline week-3-practise/level-3/1-practise-all.js]: @Ahmet-K

**Pull request closed without merging**